orchagent/aclorch.cpp

```diff
@@ -1032,6 +1096,17 @@ bool AclRule::validateAddMatch(string attr_name, string attr_value)
                 return false;
             }
         }
+        else if (attr_name == MATCH_METADATA)
+        {
+            matchData.data.u8 = to_uint<uint8_t>(attr_value);
+            matchData.mask.u8 = 0xFF;
+            // value must be between METADATA_VALUE_START and METADATA_VALUE_END inclusive.
+            if (matchData.data.u8  < METADATA_VALUE_START || matchData.data.u8 > METADATA_VALUE_END)
+            {
+                SWSS_LOG_ERROR("Invalid MATCH_METADATA configuration: %s, expected value between 1-127", attr_value.c_str());
+                return false;
```

Comment:
The error message doesn't align with the value of `METADATA_VALUE_END`.

Reply:
fixed